tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# The Dockerfile's resulting image is used in executing Kepler Operator e2e tests on ProwCI
+FROM golang:1.20
+
+WORKDIR /workspace
+
+RUN apt-get update
```

Comment:
I think we can remove this since we aren't installing any package. Having this step has the downside that the image is unnecessarily bloated and the layer can't be cached.